src/VennDiagram/VennSeries.tsx

```diff
@@ -121,12 +120,19 @@ export const VennSeries: FC<Partial<VennSeriesProps>> = ({
         actives.includes(key) ||
         (actives.length > 0 ? null : false);
 
-      // Get the colors for the stroke
-      const stroke =
-        typeof arc.props.stroke === 'function'
-          ? // @ts-ignore
-          arc.props.stroke(data, index, isActive, isHovered)
-          : arc.props.stroke;
+      const getStrokeColor = () => {
+        if (typeof arc.props.stroke === 'function') {
+          return arc.props.stroke(data, index, isActive, isHovered);
+        }
+        if (arc.props.stroke !== undefined) {
+          return arc.props.stroke;
+        }
+
+        // fallback to fill (determined by colorScheme) if stroke is undefined
+        return fill;
+      };
+
+      const stroke = getStrokeColor();
```

Comment:
I pulled this into a function cuz it was about to be some nested ternaries which is hard to read

This will set the stroke color to default to the color scheme fill if a transparent fill is passed in

Reply:
Otherwise passing in `fill: transparent` will also set a transparent stroke, so a completely invisible venn diagram. This behavior seems more intuitive